content/docs/context.md

```diff
@@ -155,27 +156,27 @@ class MyClass extends React.Component {
   }
   render() {
     let value = this.context;
-    /* render something based on the value of MyContext */
+    /* Mycontext değerini esas alarak bir şey oluşturma */
   }
 }
 MyClass.contextType = MyContext;
 ```
 
-The `contextType` property on a class can be assigned a Context object created by [`React.createContext()`](#reactcreatecontext). This lets you consume the nearest current value of that Context type using `this.context`. You can reference this in any of the lifecycle methods including the render function.
+Bir sınıftaki `contextType` özelliğine [`React.createContext()`](#reactcreatecontext) tarafından oluşturulan bir Context nesnesi atanabilir. Bu, `this.context` 'i kullanarak bu Context türünün en yakın mevcut değerini kullanmanızı sağlar. Bunu render methodu da dahil olmak üzere yaşam döngüsü methodlarından herhangi birinde belirtebilirsiniz. 
 
-> Note:
+> Not:
 >
-> You can only subscribe to a single context using this API. If you need to read more than one see [Consuming Multiple Contexts](#consuming-multiple-contexts).
+> Bu API'yi kullanarak yalnızca tek bir içeriğe abone olabilirsiniz. Daha fazla okumanız gerekiyorsa, bakınız [Çoklu Context Tüketimi](#consuming-multiple-contexts).
 >
-> If you are using the experimental [public class fields syntax](https://babeljs.io/docs/plugins/transform-class-properties/), you can use a **static** class field to initialize your `contextType`.
+> Deneysel [genel sınıf alanları sözdizimini](https://babeljs.io/docs/plugins/transform-class-properties/) kullanıyorsanız, `contextType'ınızı` başlatmak için **statik** bir sınıf alanı kullanablirsiniz.
```

Comment:
I believe we shouldn't translate `public` and we can keep the original one in parens:

> [public sınıf alanları sözdizimini (public class fields)](....)